src/pretix/api/serializers/event.py

```diff
@@ -202,6 +202,87 @@ class Meta:
                   'presale_start', 'presale_end', 'location', 'event',
                   'item_price_overrides', 'variation_price_overrides', 'meta_data')
 
+    def validate(self, data):
+        data = super().validate(data)
+
+        full_data = self.to_internal_value(self.to_representation(self.instance)) if self.instance else {}
+        full_data.update(data)
```

Comment:
It's not a security issue in this case, but just to be safe we should validate ``item_price_overrides`` and ``variation_price_overrides`` and check that all IDs are actually items *within the correct event*. You can look at quotas for inspiration.